src/globals.h

```diff
@@ -534,7 +534,15 @@ EXTERN int	garbage_collect_at_exit INIT(= FALSE);
 #define t_super			(static_types[80])
 #define t_const_super		(static_types[81])
 
-EXTERN type_T static_types[82]
+// ================= taken from the instanceof patch
+// TODO: should this be t_object_null
```

Comment:
We can keep the below definitions instead of renaming this to t_object_null.

Reply:
OK. I don't really grok what these static types are all about and how they're used. So any a VAR_OBJECT with type where `tt_class == NULL` is a null object.